Sources/InlineSnapshotTesting/AssertInlineSnapshot.swift

```diff
@@ -70,15 +70,29 @@ import Foundation
               loaded. If a timeout is unavoidable, consider setting the "timeout" parameter of
               "assertInlineSnapshot" to a higher value.
               """,
-              file: file,
-              line: line
+              fileID: file,
```

Comment:
According to the header docs `fileID: String` should be a fileID see https://developer.apple.com/documentation/swift/fileid() which is made up of the module name and filename. It can be generated by calling `fileID()` global function from within a macro.

Reply:
That would technically be a breaking change and so it will have to be considered more. The test failures created by passing `#file` seem to be ok and so may serve as a stopgap until we are willing to make the breaking change.

Reply:
Are there any side effects in terms of where swift testing will show the error location without a valid fileID string, will it show the error at the right point in the source files?